src/python_inspector/utils_pypi.py

```diff
@@ -197,6 +201,7 @@ def download_wheel(
     repos=tuple(),
     verbose=False,
     echo_func=None,
+    python_version="3.8",
```

Comment:
Should this become a global constant instead like `DEFAULT_PYTHON_VERSION="3.8"`  likely in `python_inspector/__init__.py`